README.md

```diff
@@ -21,6 +21,17 @@ $ npm install --save resin-register-device
 Documentation
 -------------
 
+Resin-Register-Device exports a factory function, which must be called with a dependencies object, containing a configured [`resin-request`](https://github.com/resin-io-modules/resin-request) instance.
+
+Example:
+```coffee
+deviceRegister = require('resin-register-device')({
+	request: require('resin-request')({
+		dataDirectory: '/opt/cache/resin'
```

Comment:
Sounds true, this specific module doesn't need it, but currently the need for this directory is deep into the modules tree as Tim correctly expressed. Is it a big deal? The supervisor can always provide it with /tmp/resin which shouldn't be a big deal, right?